src/SFA.DAS.ApplyService.Application/Apply/Gateway/ApplicationActions/WithdrawApplicationHandler.cs

```diff
@@ -8,19 +12,39 @@ namespace SFA.DAS.ApplyService.Application.Apply.Gateway.ApplicationActions
     public class WithdrawApplicationHandler : IRequestHandler<WithdrawApplicationRequest, bool>
     {
         private readonly IApplyRepository _applyRepository;
+        private readonly IOversightReviewRepository _oversightReviewRepository;
+        private readonly IAuditService _auditService;
         private readonly ILogger<WithdrawApplicationHandler> _logger;
 
-        public WithdrawApplicationHandler(IApplyRepository applyRepository, ILogger<WithdrawApplicationHandler> logger)
+        public WithdrawApplicationHandler(IApplyRepository applyRepository, IOversightReviewRepository oversightReviewRepository, IAuditService auditService, ILogger<WithdrawApplicationHandler> logger)
         {
             _applyRepository = applyRepository;
+            _oversightReviewRepository = oversightReviewRepository;
+            _auditService = auditService;
             _logger = logger;
         }
 
         public async Task<bool> Handle(WithdrawApplicationRequest request, CancellationToken cancellationToken)
         {
             _logger.LogInformation($"Performing Withdraw Application action for ApplicationId: {request.ApplicationId}");
 
-            return await _applyRepository.WithdrawApplication(request.ApplicationId, request.Comments, request.UserId, request.UserName);
+            var oversightReview = new OversightReview
+            {
+                ApplicationId = request.ApplicationId,
+                Status = OversightReviewStatus.Withdrawn,
+                InternalComments = request.Comments,
+                UserId = request.UserId,
+                UserName = request.UserName
+            };
+
+            var result = await _applyRepository.WithdrawApplication(request.ApplicationId, request.Comments, request.UserId, request.UserName);
+
+            _auditService.StartTracking(UserAction.RecordOversightOutcome, request.UserId, request.UserName);
```

Comment:
The user action here should be the name of the use case, eg "WithdrawApplication", which you'll need to add to the enum.